### PR TITLE
[WIP] DataSummary and Min Max Value endpoint

### PR DESCRIPTION
**Pull request by joaoantunes87**
@samaxes, please review it.

### CODE REVIEW

**Comment by samaxes** [comment]
@stian-sandvold can you please add the branch `data-admin-app` in the project settings on Codacy to have it analyzed?

**Comment by stian-sandvold** [comment]
@samaxes I don't have access to the codacy that runs integrated with Github, but you can set up your own account and get the analysis yourself if you are interested. Usually we also do peer reviews before merging pull requests to dhis-core, so you can also request a review from us if you want.

**Comment by samaxes** [comment]
@joaoantunes87 looks good to merge.

File: dhis-2/dhis-web/dhis-web-apps/pom.xml [hunk 1]
```diff
@@ -39,15 +39,6 @@
...
-                <!-- Maintenance -->
-                <artifactItem>
-                  <groupId>org.hisp.dhis</groupId>
-                  <artifactId>dhis-app-maintenance</artifactId>
-                  <version>2.29-SNAPSHOT</version>
-                  <overWrite>true</overWrite>
-                  <outputDirectory>${project.build.directory}/${project.artifactId}/dhis-web-maintenance</outputDirectory>
-                  <includes>**/*.*</includes>
-                </artifactItem>
                 <!-- App management -->
```
Comment (larshelge) [inline dhis-2/dhis-web/dhis-web-apps/pom.xml]: This should not be removed I guess?

File: dhis-2/DHIS 2.0 Formatter.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<code_scheme name="DHIS 2.0 Formatter" version="173">
```
Comment (jason-p-pickering) [inline dhis-2/DHIS 2.0 Formatter.xml]: Be sure this file is not in the final commit.

**Pull request merged**